example/file_server.js

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+const http       = require('http');
+const fileServer = require('./servers/file_server');
+const argv       = require('yargs').argv;
+const colors     = require('colors');
```

Comment:
Can't see this used elsewhere.

Reply:
`uploadPath.green` uses the colors import